<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMatch</class>
 <widget class="QWidget" name="ColorMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="hSensitivityLabel">
         <property name="text">
          <string>Red Range +/-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rRangeSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>75</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="sSensitivityLabel">
         <property name="text">
          <string>Green Range +/-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gRangeSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>75</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="vSensitivityLabel">
         <property name="text">
          <string>Blue Range +/-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bRangeSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>75</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="colorRangeLabel">
       <property name="text">
        <string>Color Range:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="min_color">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="mid_color">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="max_color">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
